matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/timeline/TimelineService.kt

```diff
@@ -41,15 +41,15 @@ interface TimelineService {
      * At the opposite of getTimeLineEventLive which will be updated when local echo event is synced, it will return null in this case.
      * @param eventId the eventId to get the TimelineEvent
      */
-    fun getTimeLineEvent(eventId: String): TimelineEvent?
+    fun getTimelineEvent(eventId: String): TimelineEvent?
 
     /**
      * Creates a LiveData of Optional TimelineEvent event with eventId.
      * If the eventId is a local echo eventId, it will make the LiveData be updated with the synced TimelineEvent when coming through the sync.
      * In this case, makes sure to use the new synced eventId from the TimelineEvent class if you want to interact, as the local echo is removed from the SDK.
      * @param eventId the eventId to listen for TimelineEvent
      */
-    fun getTimeLineEventLive(eventId: String): LiveData<Optional<TimelineEvent>>
+    fun getTimelineEventLive(eventId: String): LiveData<Optional<TimelineEvent>>
```

Comment:
Can you add a file `5330.sdk` to announce the 2 API name changes?

Reply:
done